backend/grants/tests/test_admin.py

```diff
@@ -11,12 +11,44 @@
     send_voucher_via_email,
 )
 from grants.models import Grant
+from api.cms.tests.factories import GenericPageFactory
+from django.http import HttpRequest
+from wagtail.models import Site, Page
+
+from django.test.utils import override_settings
 
 pytestmark = pytest.mark.django_db
 
 
+@pytest.fixture(scope="session", autouse=True)
+def test_settings():
```

Comment:
what is this?

Reply:
I think you should just change the setting in test.py if you really need it?

But isn't ALLOWED_HOSTS ignored in DEBUG mode? I don't remember

Reply:
Was a try, removed